examples/blas/compile_time_dispatching/level3/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+#===============================================================================
+# Copyright 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+#
+# SPDX-License-Identifier: Apache-2.0
+#===============================================================================
+
+#Build object from all sources
+set(BLAS_CT_SOURCES "")
+if(ENABLE_MKLCPU_BACKEND AND ENABLE_CUBLAS_BACKEND)
+  list(APPEND BLAS_CT_SOURCES "gemm_usm_mklcpu_cublas")
```

Comment:
Does it make sense to mention the requirement about what backends should be enabled in order to make compile time examples working in the README?

Reply:
I think the idea of showing two completely different vendors in one example is good, for Intel CPU and Intel GPU we already have example in Intel oneMKL package :)